spec/features/show_a_document_with_no_creator_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.feature "Showing a document summary" do
+  scenario "User views a document that has no creator" do
+    given_there_is_a_document_with_no_creator
+    when_i_visit_the_document_page
+    the_document_creator_is_shown_as_unknown
+  end
+
+  def given_there_is_a_document_with_no_creator
+    @document = create(:document, creator_id: nil)
+  end
+
+  def when_i_visit_the_document_page
+    visit document_path(@document)
+  end
+
+  def the_document_creator_is_shown_as_unknown
+    expect(page).to have_content(I18n.t("documents.show.metadata.unknown_data"))
```

Comment:
This one could be a bit flakey, as there may be other 'Unknown's on the page in future. Could we make the copy include something about users, so the test should always pass for the right reason?

Reply:
Good point, I could add the `documents.show.metadata.created_by` microcopy as I did in the other test? What do you think?